Version.cmake

```diff
@@ -0,0 +1,19 @@
+#++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+#                          Futility Development Group                          !
+#                             All rights reserved.                             !
+#                                                                              !
+# Futility is a jointly-maintained, open-source project between the University !
+# of Michigan and Oak Ridge National Laboratory.  The copyright and license    !
+# can be found in LICENSE.txt in the head directory of this repository.        !
+#++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+
+SET(Futility_VERSION_MAJOR 2)
```

Comment:
Since the two are so closely linked, I would say keeping their versions in sync makes sense. Whenever we want to tag a new MPACT release, we can tag the corresponding Futility release at the same version.